src/pages/tags/index.astro

```diff
@@ -2,22 +2,59 @@
 import { getCollection } from "astro:content";
 
 import Layout from "@/layouts/base.astro";
+import { getAuthorOrDefault } from "@/modules/author";
 
 // Fetch all documents from the 'tags' collection
 const tags = await getCollection("tags");
 ---
 
-<Layout class="p-6">
-    <div class="space-y-4">
+<content>
+    <title>RBX docs</title>
+    <meta content="Rbx Docs" property="og:title" />
+    <meta content="Tags helpful for roblox development" property="og:description" />
+    <meta content={`/RbxDocs/tags`} property="og:url" />
+    <meta content="#4c51bf" data-react-helmet="true" name="theme-color" />
+</content>
+
+<Layout class="px-4 py-8 md:px-16 lg:px-72">
+    <div class="elements">
         {
-            tags.map(doc => (
-                <a
-                    href={`/RbxDocs/tags/${doc.slug}`}
-                    class="inline-block rounded bg-blue-500 px-4 py-2 text-white hover:bg-blue-600"
+            tags.map(async doc => (
+                <div
+                    style={{
+                        width: "22rem",
+                        height: "12.5rem",
+                        minWidth: "300px",
+                        minHeight: "200px",
+                    }}
```

Comment:
There should be valid Tailwind classes shown in the [Tailwind documentation](https://tailwindcss.com/docs/installation), in the case where there isn't you can use Tailwind's [arbitrary values syntax](https://tailwindcss.com/docs/width#arbitrary-values) for one-off values, or [extend Tailwind](https://tailwindcss.com/docs/width#using-custom-values) for frequent values.